developer/src/tike/child/UfrmDebug.pas

```diff
@@ -889,8 +868,6 @@     TMemoSelectionState = record
         else msg.Msg := WM_KEYUP;
       memo.Dispatch(msg);
 
-      DoHandleShortcut(LOBYTE(dwData));
```

Comment:
> With the move to the richedit control, hotkeys such as Ctrl+V are natively managed and we no longer need to handle them separately

Could we document this in a comment? (for maintenance)

Reply:
Is this PR sufficient? We aren't going to move away from RichEdit until we rewrite TIKE as HTML (in which case this change will be irrelevant anyway!)